### PR TITLE
README

### PR DESCRIPTION
**Pull request by linusha**
Closes https://github.com/hpi-swa-lab/BP2020RH1/issues/103

### CODE REVIEW

File: lively.collab/README.md [hunk 1]
```diff
@@ -1,3 +1,47 @@
-# Comments
+# lively.collab
 
-No description for package Comments yet.
+`lively.collab` is a collection of tools for collaboration in `lively.next`.
+
+## Comments
+
+It provides functionality to comment on any morph in a world.
```
Comment (T4rikA) [inline lively.collab/README.md]:
```suggestion
It provides functionality to comment on any morph in the world. 
```

File: lively.collab/README.md [hunk 2]
```diff
@@ -1,3 +1,47 @@
...
+
+To add a comment to a morph open the morph menu on it (either via right-click or through the menu-halo) and choose "Add comment".
+
+### The Comment Browser
+
+Most of the functionality of comments is available through the Comment Browser that can be opened via the topbar.
+
+![Topbar with highlighted comment browser button](./img/topbar.png)
+
+The browser displays all comments currently existing in the world grouped after the morph the comment belongs to. Clicking on a comment will highlight the morph it belongs to in the world.
```
Comment (frcroth) [inline lively.collab/README.md]:
```suggestion
The browser displays all comments currently existing in the world grouped by the morph the comment belongs to. Clicking on a comment will highlight the morph it belongs to in the world. 
```

**Review by T4rikA — APPROVED** [review]

File: lively.collab/README.md (hunk 1 above)
Comment (frcroth) [inline lively.collab/README.md]:
```suggestion
It provides functionality to comment on any morph in a world. Comments are objects that include a text, a timestamp, a relative morph position and the user that created it.
```

File: lively.collab/README.md [hunk 3]
```diff
@@ -1,3 +1,47 @@
...
+
+#### Resolving Comments
+
+By clicking the "Resolve" button for a comment it can marked as such. Resolved comments will be shown in the corresponding tab of the browser. When that tab is activated the indicators of the resolved comments whill be shown and their color is blue. In this tab comments can also be marked as unresolved again. The badge in the topbar will count the number of **unresolved comments** in the world.
+
+![Comment badge](./img/badge.png)
+
+### Programmatically Interacting With Comments
+
+The above described way to interact with the comments via GUI is the only supported way to interact with comments. That means programmatically interacting with them (e.g., removing comments through code) may get your world in an inconsistent state and break  things!
```
Comment (Paula-Kli) [inline lively.collab/README.md]: between break and things are two spaces

**Review by Paula-Kli — APPROVED** [review]

**Pull request merged**